text/0000-command-ergonomics.md

```diff
@@ -560,8 +560,8 @@ impl Command {
 }
 ```
 
-(It is not sensibly possible at least on Unix
-to get all of the stdout and stderr output
+(It [can be difficult or impossible](https://docs.rs/io-mux/latest/io_mux/) at least on Unix
```

Comment:
(I don't actually need this cited, it's off-topic for this RFC, I just didn't want it described as impossible. :) )

Reply:
I know you weren't wanting a plug but I like to give references :-).  If I had been the reader rather than the author, I would have appreciated that link which is why I put it in.  I can take it out if it's not appropriate...

Reply:
Perfectly fine, I just wanted to make sure it hadn't come across like I was asking for a plug.